### DISCUSSION TITLE
How do we get the call history of the iPhone device that is connected?

### DISCUSSION THREAD

**Discussion by Elanchezhiyan-P** [artehe/Netimobiledevice#38] [Q&A]
Hi @artehe , 

I tried to get the call history of the iPhone device. I have a few questions related to this topic.

1. Is the call history stored in the SQLite database?
2. If stored, can we get the path of the SQLite database that is present in the iPhone?
3. If not stored in the database, How do we get the call history from the connected iPhone?

**Comment by artehe** [comment]
Answered I think in #58
